### PR TITLE
Some additions to sirius.tokenfield and sirius.autocomplete

### PR DESCRIPTION
**Pull request by welzfabian**

### CODE REVIEW

File: src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta [hunk 1]
```diff
@@ -278,6 +275,10 @@
 
                 clearTokens: function () {
                     input.clearTokens();
+                },
+
+                removeToken: function (token) {
```
Comment (welzfabian) [inline src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta]: should be in next commit

File: src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta [hunk 2]
```diff
@@ -61,9 +61,6 @@
                     input.updateTokenRemovalVisibility();
 
                     this.element.on('tokenfield:removedtoken', function (e) {
-                        var identifier = e.attrs.value;
-                        $('input[data-identifier="' + identifier + '"]').remove();
-
```
Comment (welzfabian) [inline src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta]: i can't see why we do this, but I may miss something

File: src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta [hunk 3]
```diff
@@ -53,9 +53,6 @@
                         this.tokenRemovalElement.insertAfter(this.tokenElement);
                     }
 
-                    // delete the element value because they are tokens now and the value should not be displayed twice
-                    this.tokenElement.val("");
```
Comment (welzfabian) [inline src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta]: I can't see why we do this, but I maybe missed something

File: src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta (hunk 2 above)
Comment (welzfabian) [reply src/main/resources/default/templates/wondergem/scripts/tokenfield.html.pasta]: I believe this were hidden inputs which where added to be available in posts. Theses were probably removed again.

I reimplemented the hidden inputs in https://github.com/scireum/sirius-web/pull/540/commits/d3acd32c5ca7e757901b18c26a1907b571a3035d

**Comment by sabieber** [comment]
Some JSDoc for public API methods and event listeners would be highly appreciated

**Pull request merged**